Detectors/Calibration/src/MeanVertexCalibrator.cxx

```diff
@@ -236,13 +259,17 @@ void MeanVertexCalibrator::fitMeanVertex(o2::calibration::MeanVertexData* c, Mea
     LOG(info) << "SigmaX for MeanVertex = " << sigmaX;
     LOG(info) << "SigmaY for MeanVertex = " << sigmaY;
   }
+  if (sigmaX == 0 || sigmaY == 0 || mvo.getSigmaZ() == 0 || nBinsOK < 2) {
```

Comment:
@shahor02 To be on the safe side would it not be better to do the same check as in the primary vertexer? So test for `sigmaX * sigmaX * sigmaY * sigmaY < PVertexer::kAlmost0D`?

Reply:
Ok, thanks for explaining. Then we can reinclude the mean vertex calibration after the next software update